tt_metal/hw/inc/blackhole/dev_mem_map.h

```diff
@@ -29,7 +29,8 @@
 #define MEM_L1_SIZE (1464 * 1024)
 
 #define MEM_ETH_BASE 0x0
-#define MEM_ETH_SIZE (256 * 1024)
+// -32 for ETH barrier, see comment in eth_l1_address_map
+#define MEM_ETH_SIZE (256 * 1024 - 32)
```

Comment:
I will update this MEM_ETH_SIZE to be (512 * 1024 - 32) in my PR to enable FD out of idle erisc cores (in this file and everywhere else where it is incorrect)